patient-portal-ui/src/pages/patient/PatientExamDetails.tsx

```diff
@@ -53,88 +51,86 @@ function PatientExamDetails(props: {
         </Grid>
         <Grid xs={6}>
           <Typography style={{ fontWeight: 600 }}>
-            Reparto
+            Date
           </Typography>
           <Typography display="block" variant="button" gutterBottom>
-            {props.setType.r_adm_wrd_id_a_desc}
+            {props.setType.r_lab_date_date}
           </Typography>
         </Grid>
         <Grid xs={6}>
           <Typography style={{ fontWeight: 600 }}>
-            Tipo Ricovero
+            Time
           </Typography>
           <Typography display="block" variant="button" gutterBottom>
-            {props.setType.r_adm_admt_id_a_adm_desc}
+            {props.setType.r_lab_date_hour}
           </Typography>
         </Grid>
         <Grid xs={6}>
           <Typography style={{ fontWeight: 600 }}>
-            Ricovero
+            Categoria Esame
```

Comment:
Let's use English translations all over the code

Reply:
ok, I'll change in the comments too